classes/Table/OpencastEventListTableGUI.php

```diff
@@ -153,8 +148,8 @@ protected function fillRow($row): void
             $this->tpl->setVariable('DATA_TITLE', $row['title']);
             $this->tpl->setVariable('DATA_DESCRIPTION', $row['description']);
 
-            $this->tpl->setVariable('UNSELECTED_ICON', ilUtil::getHtmlPath(ilUtil::getImagePath("radiobutton_unchecked.png")));
-            $this->tpl->setVariable('SELECTED_ICON', ilUtil::getHtmlPath(ilUtil::getImagePath("radiobutton_checked.png")));
+            $this->tpl->setVariable('UNSELECTED_ICON', $this->plugin->getDirectory() . '/templates/images/circle.svg');
```

Comment:
Unfortunately, this is not quite the same as before using the method from ilUtil: The old method - if available and activated for the current user - takes the image from a custom skin instead of the standard skin (Delos). With this change, the same image would now always be used by the plugin. But I can't say whether you want that, I'll leave that to others.

Reply:
That was intended, because, there is no proper replacement for the check/uncheck icon to provide from ILIAS itself.